src/pages/amazon-sales-channel/best-practices/logging-troubleshooting.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Logging and troubleshooting
+description: Learn the development team's recommendations about logging, based on their experience developing the Amazon Sales Channel app. Troubleshooting tips are also provided. 
+---
+
+# Logging and troubleshooting
+
+Adobe Developer App Builder offers different options to facilitate logging from code and for viewing and forwarding applications logs as needed. You can opt for the default setting, which involves retrieving logs directly from the IO Runtime, or you can forward them to other destinations like Splunk or Azure Log Analytics. This topic describes implementing the default option and forwarding to Splunk.
+
+## Adobe I/O Runtime (default)
+
+[Actions](https://developer.adobe.com/runtime/docs/guides/overview/entities/) are serverless functions that run on Adobe I/O Runtime. In general, an action is invoked in response to an event and produces some observable output. An Adobe Commerce event must work successfully for its resulting Runtime Action to be successful. Make sure to troubleshoot both of these entities when diagnosing issues with your application. The following tips and comprehensive comparison enable you to make informed decisions about your Logging approach.
+
+When building and troubleshooting your application, it is important to understand the differences between events and runtime actions. These concepts are often interrelated, with events triggering runtime actions, but the actual functionality and logging capabilities are separate entities. Best practice is to determine whether the event delivery was successful, and then investigate any resulting runtime actions.
+
+### Set the log level
+
+You can configure the log level per action in your `ext.config.yml` or `app.config.yml` file, as follows:
+
+```yaml
+<action_name>:
+  inputs:
+    LOG_LEVEL: <error|warn|info|verbose|debug|silly>
+```
+
+### View logs
+
+To view logs in real time, run the following command:
+
+```bash
+aio rt logs --tail
+```
+
+```terminal
+283976-asctestamina-stage/amazon-app/scheduled-amazon-sync-products:e7e5281de72049d3a5281de720c9d3c7
+2023-07-18T10:27:05.585Z       stdout: 2023-07-18T10:27:05.585Z [main /283976-asctestamina-stage/amazon-app/scheduled-amazon-sync-products] info: {"message":"Start retrieving Amazon product updates","params":{}}
+```
+
+Sometimes logs do not appear, like web action logs or small random hiccups. In such a case, you can use the activation ID to get specific logs for that action.
```

Comment:
Hi Kevin, looking good, can we add something like this? 

When an action is invoked, it produces an activation record that contains information about this invocation, including activationId , logs , statusCode , and more. You can find [a full list of the properties of activation here](https://github.com/apache/openwhisk/blob/master/docs/actions.md#understanding-the-activation-record). By default, activations have a 7-day TTL. Activation IDs are an excellent resource to key on for debugging internally and should be provided in the description of support issues around runtime actions.